assets/sass/components/thank-with-google/_googlesitekit-twg-color-radio.scss

```diff
@@ -45,7 +46,7 @@
 				}
 
 				.mdc-image-radio__content[aria-checked="true"] {
-					outline: 0.1875rem solid;
```

Comment:
Why are we using rem here when the other styles use px? 🤔 

Also, this still looks off when compared with the design.

Left Figma @ 100%, Right is this PR (Discoloration is due to the shadow between overlapping windows)
![image](https://user-images.githubusercontent.com/1621608/191839994-f4333dcd-6ff3-4da8-be64-e1a4b8c47f88.png)

Here it is using 3px for the outline instead which looks right to me 🙂 
![image](https://user-images.githubusercontent.com/1621608/191840072-649ebcd5-a2cd-4bfe-b495-1d78dd9a2694.png)

Looking a bit closer this appears to be defined in Figma too, but there appears to be overlapping outlines of different sizes depending on what is selected :

![image](https://user-images.githubusercontent.com/1621608/191840832-06ec391a-46dc-429c-b5af-0e00c40567cf.png)
![image](https://user-images.githubusercontent.com/1621608/191840873-bbf06226-ad0a-428c-9fdd-5b49c71715f9.png)

Let's go with the 3px outline instead 😄 
(this is also consistent with the outline thickness of the image radios as well)